src/renderer/atlas/pch.h

```diff
@@ -50,5 +50,9 @@
 #include <base/numerics/safe_math.h>
 #pragma warning(pop)
 
+// {fmt}, a C++20-compatible formatting library
+#include <fmt/format.h>

```

Comment:
y atlas need

Reply:
The non-templated functions in the header files which use `fmt` fail to compile otherwise.